rules/rule_defs.go

```diff
@@ -57,9 +57,10 @@ const (
 	IPSetIDNATOutgoingAllPools  = "all-ipam-pools"
 	IPSetIDNATOutgoingMasqPools = "masq-ipam-pools"
 
-	IPSetIDAllHostNets        = "all-hosts-net"
-	IPSetIDAllVXLANSourceNets = "all-vxlan-net"
-	IPSetIDThisHostIPs        = "this-host"
+	IPSetIDAllHostNets          = "all-hosts-net"
+	IPSetIDAllWireguardHostNets = "all-wg-hosts-net"
```

Comment:
Don't think you used this in the end?

Reply:
Ah no - I will remove.  Cruft from my cherry-pick from the other PR